### PR TITLE
feat(user): add `user/me` endpoint

### PR DESCRIPTION
**Pull request by Khaledgarbaya**
This PR adds the `/user/me/` endpoint to be able to get the currently authenticated user.

- [x] Unit test
- [x] Integration test ??

### CODE REVIEW

File: lib/create-contentful-api.js [hunk 1]
```diff
@@ -41,7 +41,7 @@
 import { createRequestConfig } from 'contentful-sdk-core'
 import errorHandler from './error-handler'
 import entities from './entities'
-
+import {wrapUser} from './entities/user'
```
Comment (axe312ger) [inline lib/create-contentful-api.js]: so this was the solution to the import problem? :(

I have a bad feeling with having the entities imported twice, one general object with all entities as property plus wrapUser from the user separate? 👎

Comment (axe312ger) [reply lib/create-contentful-api.js]: Should be at the beginning of `createClientApi` as `wrapSpace` and `wrapOrganizationCollection`.

I can give it a try? 🤷‍♂️

Comment (Khaledgarbaya) [reply lib/create-contentful-api.js]: I won't merge the PR yet until I find out what is going on with the entities

**Comment by axe312ger** [comment]
We should add tests for https://coveralls.io/builds/13334564/source?filename=lib%2Fcreate-contentful-api.js#L156

**Comment by axe312ger** [comment]
But shouldn't https://github.com/contentful/contentful-management.js/pull/139/commits/2698e448659ece3a690291ddce2c625eca07f51d#diff-f8a01c6833d232cf4efc7d4e518a1d31 cover it? 🤔

**Comment by Khaledgarbaya** [comment]
That was integration test

**Comment by axe312ger** [comment]
@Khaledgarbaya rebase/squash and then commit? LGTM

**Pull request merged**